### DISCUSSION TITLE
Microsoft Store App (new) - do win32 apps (auto) update like UWP apps?

### DISCUSSION THREAD

**Discussion by SanderVerroen** [microsoft/winget-cli#2875] [Q&A] [answered]
Within Intune a new type of app 'Microsoft Store app (new)' can be deployed. Most apps are UWP apps but some are Win32.

Do these win32 apps update themselves in the same way as UWP apps, though Windows update?
Are these apps updated through another mechanism, if so, how?
If they are not updated, will this be a feature in the future?

I know I can update apps in a variety of ways, inclusing 'winget --upgrade' but these are manual/scripted solutions. And yes, I am aware that the win32 type is still experimental ;)

**Comment by dongle-the-gadget** [comment, marked as answer]
Depends on the Win32 app itself. If it's a MSIX packaged application, then it is updated via Windows Update. If it's the EXE/MSI install model, the application is fully responsible for upgrading itself.
It may be worthy to know that Microsoft Store also uses the Windows Package Manager API that winget does.

**Reply by xhoy** [reply]
Is it possible to see what kind of package is listed in the MS store? Aka can i tell as a sys admin what kind of package it is?

**Reply by dongle-the-gadget** [reply]
Look for the "Provided and updated by" tag under the app name in the app listing page. If it shows up, it means that the app is an unpackaged Win32 listing.
